common/app/views/fragments/atoms/storyquestions.scala.html

```diff
@@ -0,0 +1,26 @@
+@(storyquestions: model.content.StoryQuestionsAtom, isAmp: Boolean)(implicit request: RequestHeader)
+
+@if(!isAmp) {
+    <div class="submeta user__question">
+    <span class="js-storyquestion-atom-id" style="display: none" id="@storyquestions.id"></span>
```

Comment:
Could you use the `is-hidden` class instead https://github.com/guardian/frontend/blob/master/static/src/stylesheets/base/_helpers.scss#L63